### PR TITLE
data: STIAANWOL

### PR DESCRIPTION
**Pull request by STIAANWOL**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5582"></a>

### CODE REVIEW

File: data/STIAANWOL.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Stiaan Wolfaardt",
+  "type": "personal",
+  "bio": "Software Engineer based in Cape Town, South Africa.",
+  "avatar": "https://github.com/STIAANWOL.png",
```
Comment (khushalarora) [inline data/STIAANWOL.json:5]: Hi @STIAANWOL,  thank you for contributing  :tada: 
You can delete this line, it is not required anymore :+1: 
```Suggestion
```

**Review by khushalarora — CHANGES_REQUESTED** [review]

**Comment by STIAANWOL** [comment]
@khushalarora Changes applied. Thank you.

**Review by khushalarora — APPROVED** [review]
Looks good, thanks  :tada:

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for your Contribution! 🎉 

<p align="center"></p>

**Pull request merged**